tools/internal_ci/linux/grpc_tsan_on_foundry.sh

```diff
@@ -38,7 +38,7 @@ source tools/internal_ci/helper_scripts/prepare_build_linux_rc
 "${KOKORO_GFILE_DIR}/bazel_wrapper.py" \
   --host_jvm_args=-Dbazel.DigestFunction=SHA1 \
   test --jobs="50" \
-  --test_timeout="300,450,1200,3600" \
+  --test_timeout="1500,1500,1500,3600" \
```

Comment:
Just curious: how is the test_timeout argument formatted?

Reply:
They correspond to short, moderate, long and external tests.